docs/reST/ref/joystick.rst

```diff
@@ -500,9 +545,11 @@ In pygame 1 the X360 controller mapping has 5 Axes, 10 buttons and 1 hat.
     Left -> Right   - X Axis
 
 
-**Playstation 4 Controller (name: "Wireless Controller")**
+Playstation 4 Controller (pygame 1.x)
+*************************************
 
-In pygame 1 the PS4 controller mapping has 6 Axes and 14 buttons and 1 hat.
+The PlayStation 4 controller mapping has 6 axes, 14 buttons, and 1 hat.
+name: "Wireless Controller"
 
 * **Left Stick**::
```

Comment:
I don't think pygame 1.x info is particularly relevant here, this might need updating (perhaps in a future PR)

Reply:
It's relevant to users of the library who want to support pygame 1.9.x and 2.x. No?

Reply:
yeah, true.

Looks like I was having a brain fart when I reviewed this 😅